cmake/showoptions.cmake

```
@@ -12,6 +12,15 @@ message("* Install core to        : ${CMAKE_INSTALL_PREFIX}")
 if( UNIX )
   message("* Install configs to     : ${CONF_DIR}")
 endif()
+
+if ( COPY_CONF )
```

Just small nit-pick (sorry!): we don't use spaces around parentheses in CMake
```suggestion
if( COPY_CONF )
```